2200040/문자열의 거울상.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+sys.stdin = open('input.txt', 'r')
+
+T = int(input())
+# 케이스 수
+
+for t in range(1, T+1):
+    ts = input()
+    ts = list(reversed(ts))
+    cnt = 0
+    for i in ts:
+        if i =='q':
+            ts[cnt] = 'p'
+            cnt += 1
+        elif i == 'p':
+            ts[cnt] ='q'
+            cnt += 1
+        elif i == 'd':
+            ts[cnt] = 'b'
+            cnt += 1
+        elif i == 'b':
+            ts[cnt] = 'd'
+            cnt += 1
```

Comment:
```suggestion
    for cnt, i in enumerate(ts):
        if i =='q':
            ts[cnt] = 'p'
        elif i == 'p':
            ts[cnt] ='q'
        elif i == 'd':
            ts[cnt] = 'b'
        elif i == 'b':
            ts[cnt] = 'd'
```
좋은 풀이네요!
추가로, python의 내장 함수 중 `enumerate`라는 함수가 있습니다.
이 함수를 사용하시면 `cnt`를 따로 더 조작하지 않으면서 더 깔끔한 코드가 될 것 같습니다. 😄